### ISSUE TITLE
Update F&E accessibility statement

### ISSUE THREAD

**Issue by dgamblin3008** [DFE-Digital/npd-find-and-explore#609]
**INFO**
An accessibility assurance re-test of Find & Explore was carried out on 14/09/2020.  The assessment found no issues and the report concluded that the service meets the WCAG 2.1 AA level of compliance.

**CHANGES TO ACCESSIBILITY STATEMENT**
We therefore need to make two changes to the F&E accessibility statement (https://find-npd-data.education.gov.uk/accessibility):

1.  There's a statement under 'How accessible this service is' that says, 'we don't currently have filters on our search. We're working on including filters as part of our next release which should help manage the amount of data a search could return'. This statement is out of date as filters are provided for search results, so should be removed. 

2.  Under 'How we tested this service', the date of the test should be changed from ‘01 September 2019’ to ‘14 September 2020’.

**Comment by dgamblin3008** [comment]
Tested this on staging. Moving to 'Ready for Release'

**Comment by dgamblin3008** [comment]
Deployed to production and tested successfully. Awaiting approval by PO, Gwen McGill, before closing the ticket.

**Comment by dgamblin3008** [comment]
PO has agreed that issue can be closed.
